engine/src/tangl/media/media_creators/dicebear_forge/dicebear_spec.py

```
@@ -0,0 +1,182 @@
+    style_definition_version: str = DICEBEAR_STYLE_VERSION
+    style_definition_hash: str
+    seed: str
+    options: dict[str, Any]
+    adapter_version: str = DICEBEAR_ADAPTER_VERSION
+    ignored_traits: dict[str, str] = Field(default_factory=dict)
+    renderer_name: str | None = None
+    renderer_version: str | None = None
+    resolved_options: dict[str, Any] | None = None
+
+    @classmethod
+    def get_creation_service(cls) -> "DiceBearForge":
```

Ruff (0.16.0)

[error] Undefined name `DiceBearForge`

(F821)